Makefile

```diff
@@ -1,12 +1,16 @@
-TEST_TARGET := ./tests/
-PACKIT_IMAGE := docker.io/usercont/packit-service:master
+SERVICE_IMAGE := docker.io/usercont/packit-service
+WORKER_IMAGE := docker.io/usercont/packit-service-worker
```

Comment:
When the image described in `Dockerfile.worker.prod` is built? Is it just docker hub hook?

Reply:
Yes, it's just for docker hub trigger on a push to stable branch.
I haven't figured out any better way.